packages/dev/loaders/src/BVH/bvhLoader.ts

```diff
@@ -8,6 +8,7 @@ import type { Scene } from "core/scene";
 import type { Nullable } from "core/types";
 import type { BVHLoadingOptions } from "./bvhLoadingOptions";
 import { Tools } from "core/Misc/tools";
+import { AssetContainer } from "core/assetContainer";
```

Comment:
```suggestion
import type { AssetContainer } from "core/assetContainer";
```

Reply:
I think the loader package should also have eslint.

Reply:
Hmm.... I will have to check what happened here. Thanks :-)

Reply:
i guess you are just not fully updated to the latest. Run `npm install` after pulling from master, it should work